substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -224,15 +236,18 @@ pub mod pallet {
 		/// An `ExposurePage` is weakly bounded to a maximum of `MaxExposurePageSize`
 		/// nominators.
 		///
-		/// For older non-paged exposure, a reward payout was restricted to the top
-		/// `MaxExposurePageSize` nominators. This is to limit the i/o cost for the
-		/// nominator payout.
-		///
 		/// Note: `MaxExposurePageSize` is used to bound `ClaimedRewards` and is unsafe to reduce
```

Comment:
is it true, I see `MaxRewardPagesPerValidator` used instead.

Reply:
Right, changed comments here and for `MaxRewardPagesPerValidator` in https://github.com/paritytech/polkadot-sdk/pull/6445/commits/c881bb0015cfd0c9248ebd53f751ff267b5ba5d9

I'm using "it may be unsafe" because we are using `WeakBoundedVec`s